tests/unit/datarobot_drum/drum/language_predictors/test_base_language_predictor.py

```diff
@@ -27,25 +27,72 @@ def _chat(self, completion_create_params):
 
 
 class TestChat:
+    @pytest.fixture
+    def language_predictor(self, chat_python_model_adapter):
+        predictor = TestLanguagePredictor()
+        predictor.mlpiper_configure(
+            {
+                "target_type": TargetType.TEXT_GENERATION,
+                "__custom_model_path__": "/non-existing-path-to-avoid-loading-unwanted-artifacts",
+            }
+        )
+
+        yield predictor
+
     @pytest.fixture
     def mock_mlops(self):
-        with patch("datarobot_drum.drum.language_predictors.base_language_predictor.MLOps") as mock:
+        with (
+            patch(
+                "datarobot_drum.drum.language_predictors.base_language_predictor.mlops_loaded", True
+            ),
+            patch("datarobot_drum.drum.language_predictors.base_language_predictor.MLOps") as mock,
+        ):
             mlops_instance = Mock()
             mock.return_value = mlops_instance
             yield mlops_instance
 
     @pytest.fixture
-    def language_predictor(self, chat_python_model_adapter, mock_mlops):
+    def language_predictor_with_mlops(self, chat_python_model_adapter, mock_mlops):
         predictor = TestLanguagePredictor()
         predictor.mlpiper_configure(
             {
                 "target_type": TargetType.TEXT_GENERATION,
                 "__custom_model_path__": "/non-existing-path-to-avoid-loading-unwanted-artifacts",
+                "deployment_id": "deployment_id",
             }
         )
 
         yield predictor
 
+    @pytest.mark.parametrize("stream", [False, True])
+    def test_chat_without_mlops(self, language_predictor, stream):
```

Comment:
I think it would be better for readability to split out the parametrized test. So you would have two tests (names are suggestions):
```
test_chat_yes_stream_without_mlops
test_chat_no_stream_without_mlops
```

Reply:
Thanks for the suggestion but I'm using the `stream` parameter in many tests in this file so I would prefer to keep it like this for consistency.